CHANGELOG.cdf-tk.md

```diff
@@ -23,7 +23,7 @@ Changes are grouped as follows:
 
 ### Changed
 
-- The `cdf-tk build` will now clean the build directory by default before building the modules. To stop this behavior,
+- [BREAKING] The `cdf-tk build` will now clean the build directory by default before building the modules. To stop this behavior,
```

Comment:
```suggestion
- [BREAKING] The `cdf-tk build` will now clean the build directory by default before building the modules to avoid unwanted side effects from previous builds. To stop this behavior,
```